From 1281b60a1807d1285b101b6eb61c6478f29785fe Mon Sep 17 00:00:00 2001
From: George Joseph <george.joseph@fairview5.com>
Date: Wed, 23 Mar 2016 07:48:52 -0600
Subject: [PATCH] aconfigure.ac:  Fix autoconf issue with opencore-amrnb on
 older systems

autoconf 2.63 on CentOS6 produces a bad ./aconfigure file related to
opencore-amrnb.

./aconfigure: line 15158: syntax error near unexpected token `fi'

To get around this, a 'true;' needed to be added to the Ok case of
AC_ARG_WITH(opencore-amrnb)
---
 aconfigure    | 3 +++
 aconfigure.ac | 2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/aconfigure b/aconfigure
index 33a08f5..4c122c2 100755
--- a/aconfigure
+++ b/aconfigure
@@ -7908,6 +7908,9 @@ fi
 # Check whether --with-opencore-amrnb was given.
 if test "${with_opencore_amrnb+set}" = set; then :
   withval=$with_opencore_amrnb; as_fn_error $? "This option is obsolete and replaced by --with-opencore-amr=DIR" "$LINENO" 5
+else
+  true;
+
 fi
 
 
diff --git a/aconfigure.ac b/aconfigure.ac
index 3e88124..5d3e833 100644
--- a/aconfigure.ac
+++ b/aconfigure.ac
@@ -1631,7 +1631,7 @@ AC_ARG_WITH(opencore-amrnb,
     AC_HELP_STRING([--with-opencore-amrnb=DIR],
 		   [This option is obsolete and replaced by --with-opencore-amr=DIR]),
     [AC_MSG_ERROR(This option is obsolete and replaced by --with-opencore-amr=DIR)],
-    []
+    [true;]
     )
 
 dnl # opencore-amr alt prefix
-- 
2.5.0

